src/core/core.tooltip.js

```diff
@@ -677,13 +677,17 @@ module.exports = function(Chart) {
 			};
 
 			// Before body lines
+			ctx.fillStyle = mergeOpacity(vm.bodyFontColor, opacity);
 			helpers.each(vm.beforeBody, fillLineOfText);
 
 			var drawColorBoxes = vm.displayColors;
 			xLinePadding = drawColorBoxes ? (bodyFontSize + 2) : 0;
 
 			// Draw body lines now
 			helpers.each(body, function(bodyItem, i) {
+				var labelTextColor = mergeOpacity(vm.labelTextColors[i], opacity);
```

Comment:
there's no need to have an intermediate variable here. the code below would work and result in a smaller file size

    ctx.fillStyle = mergeOpacity(vm.labelTextColors[i], opacity);

Reply:
Oops, sorry.  I fixed it.